Gemfile

```
@@ -8,4 +8,4 @@ gem "appraisal"
 gem "bundler"
 gem "minitest"
 gem "rake"
-gem "rails", ">= 5.0"
+gem "rails", ">= 6.0"
```

@toshimaru How about requiring Rails 5.2 or higher because Rails 5.2 is maintained?
https://guides.rubyonrails.org/maintenance_policy.html

```suggestion
gem "rails", ">= 5.2"
```

---

Okay, it was too early to drop Rails 5.x! 😲 
I will change it to v5.2. 👍